epicure-admin/src/App.tsx

```diff
@@ -0,0 +1,15 @@
+import { BrowserRouter , Routes, Route } from "react-router-dom";
+import Dashboard from "./shared/components/Dashboard";
+
+const App: React.FC = () => {
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route path="/dashboard/" element={<Dashboard />} />
+        <Route path="/dashboard/:collection" element={<Dashboard />} />
```

Comment:
- routes path  should be under const ROUTES
- if you have this moment one page you should have redirect to this page